zipline/finance/blotter.py

```diff
@@ -179,64 +185,62 @@ def process_benchmark(self, benchmark_event):
         return
         yield
 
-    def process_trade(self, trade_event):
+    def process_open_orders(self):
```

Comment:
That's actually how I did it on my first pass, but then realized that blotter has current_dt on it, so ended up using current_dt.  Thought about removing current_dt, but the various order methods use it to set the date on the new orders.

I think we should stay consistent - either blotter knows about current simulation time, or it doesn't.  Since it does right now, I went with it.